sample_size/multiple_testing.py

```diff
@@ -2,6 +2,7 @@
 
 import numpy as np
 import numpy.typing as npt
+from numpy.random import RandomState
 from statsmodels.stats.multitest import multipletests
 
 from sample_size.metrics import BaseMetric
```

Comment:
Could you define the random seed up here?

RANDOM_SEED = 1

Then we import it in `sample_size/metrics.py`?